.rubocop_todo.yml

```diff
@@ -1,41 +1,27 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-08-01 11:17:00 -0400 using RuboCop version 0.73.0.
+# on 2019-10-08 13:35:11 +0200 using RuboCop version 0.75.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 9
+# Offense count: 236
 Metrics/AbcSize:
-  Enabled: true
-  Max: 50
-  Exclude:
-    - 'app/controllers/articles_controller.rb'
-    - 'app/controllers/comments_controller.rb'
-    - 'app/controllers/follows_controller.rb'
-    - 'app/controllers/internal/articles_controller.rb'
-    - 'app/controllers/omniauth_callbacks_controller.rb'
-    - 'app/labor/hex_comparer.rb'
-    - 'app/models/tweet.rb'
-    - 'app/services/notifications/new_comment/send.rb'
-    - 'app/services/notifications/reactions/send.rb'
+  Max: 84
 
-# Offense count: 13
+# Offense count: 12
 # Configuration parameters: CountComments, ExcludedMethods.
 # ExcludedMethods: refine
 Metrics/BlockLength:
-  Max: 63
+  Max: 62
 
-# Offense count: 1
+# Offense count: 322
 # Configuration parameters: Max.
 RSpec/ExampleLength:
-  Max: 10
```

Comment:
Do we have that many violations with `Max: 10` to disable the cop?

Reply:
if you check the comment above it you can read how many violations there are, right now there are 322

Reply:
Certain request specs eat up most of the file lines purely due to verbosity of adding the appropriate parameters in a multi-line way that is compliant with other linting.

Reply:
yeah, I don't think we're going to enable `Metrics/LineLength` anytime soon, but without it we're also only adding new lines that are long :D It's a catch 22

Reply:
@rhymes there are 322 violations when there is no Max specified, so when Max is equal to default (5) which is too low imo. With the previous settings - `Max: 10` + `Exclude` we had no violations. I think it's better than disabling the cop overall.

Reply:
When it's `Enabled: false` in the todo, `rubocop` won't check the new tests with this cop.

Reply:
@lightalloy ah, got it! I'm going to move the configuration in the rubocop file, so that it's not disabled, except for those two entries in the exclude

Reply:
@lightalloy let me know if now could work